frontend/src/components/camp/Picasso.vue

```diff
@@ -121,6 +122,11 @@ export default {
     scheduleEntries: {
       type: Array,
       required: true
+    },
+    editable: {
```

Comment:
Soll das einen read only mode bewirken?
Ich habe das bis jetzt disabled genannt (weil das in html schon so ist. Ich weiss, es ist verwirrend, ich wollte das konstant behalten)

Reply:
Disabled wäre falsch, da man ja trotzdem die scheduleEntries klicken kann.